### PR TITLE
feat(registry): Add pgEdge provider package

### PR DESCRIPTION
**Pull request by KnockOutEZ**
## Description

This PR adds the pgEdge provider package to the Pulumi Registry. The pgEdge provider enables management of pgEdge Cloud resources, supporting both Developer and Enterprise editions. This provider allows users to manage databases, clusters, backup stores, and cloud accounts through Pulumi infrastructure as code.

## Adding a new package?

If this pull request adds a new package:

- [x] The package's schema URL in this PR is correct (`provider/cmd/pulumi-resource-pgedge/schema.json`).
- [x] The package metadata file contains:
  - [x] a supported category (`Database`).
  - [x] a valid plugin download URL (via GitHub repo: https://github.com/pgedge/pulumi-pgedge).
  - [x] a description ("A Pulumi package for creating and managing pgEdge Cloud resources.").
  - [x] a valid logo URL (https://pgedge-public-assets.s3.amazonaws.com/product/images/pgedge_mark.svg).
- [x] The package repo contains an Overview doc (`/docs/_index.md`) that includes:
  - [x] a brief explanation of the pgEdge provider and its capabilities.
  - [x] representative examples in TypeScript and Go.
  - [x] front-matter layout property set to `package`.
- [x] The package repo contains an Installation and Configuration doc (`/docs/installation-configuration.md`) that includes:
  - [x] links to Node.js and Go SDKs.
  - [x] installation commands for npm/yarn and go get.
  - [x] configuration examples using environment variables (`PGEDGE_CLIENT_ID` and `PGEDGE_CLIENT_SECRET`).
- [x] The repository has:
  - [x] version tag v0.0.28 with corresponding GitHub release
- [x] A CODEOWNER has reviewed the PR.
- [x] A member of the @pulumi/docs team has reviewed all documentation.

Additional Notes:
- Package is currently in public preview status
- Supports both Developer and Enterprise editions with different feature sets
- Includes comprehensive support for database, cluster, and backup management
- Provides multi-region deployment capabilities

### CODE REVIEW

File: tools/resourcedocsgen/pkg/lookup.go
Comment (iwahbe) [inline tools/resourcedocsgen/pkg/lookup.go:193]: `TitleLookup` is being deprecated in favor of the `"displayName"` field in the provider schema. If you set `tfbridge.ProviderInfo.DisplayName`, then it will flow through.

We are not adding new packages to `lookup.go`.

File: tools/resourcedocsgen/pkg/lookup.go
Comment (iwahbe) [inline tools/resourcedocsgen/pkg/lookup.go:95]: Instead of setting this in the override map, please just set `catagory/database` in your schemas `"keywords"` section with `tfbridge.ProviderInfo.Keywords`.

File: themes/default/data/registry/packages/pgedge.yaml
Comment (iwahbe) [inline themes/default/data/registry/packages/pgedge.yaml:1]: This file is auto-generated, and will be added automatically.

File: themes/default/content/registry/packages/pgedge/_index.md
Comment (iwahbe) [inline themes/default/content/registry/packages/pgedge/_index.md:1]: Docs files are pulled from your repo. This file should live at `github.com/pgedge/pulumi-pgedge/docs/_index.md`. The registry will pick it up during builds.

This way the docs are:
1. Versioned with your provider.
2. Editable by *you* without going through our PR process.

File: themes/default/content/registry/packages/pgedge/installation-configuration.md
Comment (iwahbe) [inline themes/default/content/registry/packages/pgedge/installation-configuration.md:1]: Docs files are pulled from your repo. This file should live at github.com/pgedge/pulumi-pgedge/docs/installation-configuration.md. The registry will pick it up during builds.

File: community-packages/package-list.json
Comment (iwahbe) [inline community-packages/package-list.json:1]: This is the only change we need here.

**Review by iwahbe — CHANGES_REQUESTED** [review]
Hi @KnockOutEZ, thanks for opening a PR and writing such great docs. We will need to make a couple of changes before merging.

**Comment by KnockOutEZ** [comment]
Thanks for the feedbacks @iwahbe . I just fixed the PR items and also updated the pulumi-pgedge repo.

**Comment by mjeffryes** [comment]
Thanks for the contribution @KnockOutEZ. A few questions:
- Are you the best contact to represent pgEdge as a maintainer of this provider? 
- Our policy is generally to wait until all Pulumi languages are supported before publishing in our registry, but I noticed that support for python and dotnet SDKs are listed as coming soon. Can you please publish SDKs to PyPi and Nuget and update examples in your index and configuration docs to match?

**Comment by KnockOutEZ** [comment]
Hi @mjeffryes,

Thanks for your patience! I've addressed the requested changes:

1. The SDKs for Python and .NET are now published on PyPI and NuGet. The documentation has been updated with examples to reflect these changes.
2. Regarding the maintainer contact, I ([Towhid](mailto:matt@pgedge.com)) am the maintainer. However, you can contact my supervisor, Matt, at [matt@pgedge.com](mailto:matt@pgedge.com) for any partnership-related inquiries.

Please let me know if there's anything else you'd like me to adjust or review.

**Review by mjeffryes — APPROVED** [review]
Looks good! Thanks for following up to finish this!

**Pull request merged**
